ansible/roles/ocp4-workload-workshop-admin-storage/readme.adoc

```diff
@@ -31,22 +40,16 @@
 === Deploy a Workload with the `ocp-workload` playbook [Mostly for testing]
 
 ----
-export TARGET_HOST="bastion.cc59.sandbox118.opentlc.com"
-export OCP_USERNAME="kubeadmin"
-export WORKLOAD="ocp4-workload-workshop-admin-storage"
-export GUID=cc59
 export TARGET_HOST_PASSWORD=to_be_changed
 
 # a TARGET_HOST is specified in the command line, without using an inventory file
-ansible-playbook -i ${TARGET_HOST}, ./configs/ocp-workloads/ocp-workload.yml \
-    -e"ansible_ssh_private_key_file=/home/lab-user/.ssh/id_rsa" \
-    -e"ansible_user=lab-user" \
-    -e"ocp_username=${OCP_USERNAME}" \
-    -e"ocp_workload=${WORKLOAD}" \
+ansible-playbook -i $(dig @resolver1.opendns.com A myip.opendns.com +short), ./configs/ocp-workloads/ocp-workload.yml \
+    -e"ansible_ssh_private_key_file=/home/$(whoami)/.ssh/id_rsa" \
+    -e"ansible_user=$(whoami)" \
+    -e"ocp_workload=ocp4-workload-workshop-admin-storage" \
```

Comment:
can you remove this from the PR ?
the `dig` part looks very specific to your laptop

Reply:
It's basically target_host. Maybe this is extreme but I was told to minimize variables for export.
Before it was ${TARGET_HOST}